example/Python/src/ReflexGame/ReflexGame.lf

```diff
@@ -0,0 +1,192 @@
+        self.update_graphics.send(((152,251,152), 
+                                   (0, 0, 0), 
+                                   "{}. Press any key!".format(prompt.value)))
+        self.prompt_time = get_physical_time()
+    =}
+    
+    reaction(user_input) -> another {=
+        if self.first:
+            # if the first ever key press is detected, set "another" to trigger a prompt from RandomSource
+            self.first = False
+            self.update_graphics.send(((205,92,92), 
+                                       (0, 0, 0), 
+                                       "Wait for the prompt..."))
+            
+            # ask for the first ever prompt            
+            another.set(42)
+        elif self.prompt_time == 0:
+            if self.count > 0:
+                self.update_graphics.send(((205,92,92), 
+                                           (0, 0, 0), 
+                                           "YOU CHEATED!", 
+                                           "Average response time: {:.2f} ms".format(self.total_time_in_ms / self.count)))
+            else:
+                self.update_graphics.send(((205,92,92), 
+                                           (0, 0, 0), 
+                                           "YOU CHEATED!", 
+                                           "Average response time: undefined"))
+            request_stop()
+        else:
+            time_in_ms = (get_logical_time() - self.prompt_time) // MSEC(1)
+            self.update_graphics.send(((205,92,92), 
+                                       (0, 0, 0), 
+                                       "Response time in milliseconds: {}".format(time_in_ms), 
+                                       "Wait for the prompt..."))
+            self.count += 1
+            self.total_time_in_ms += time_in_ms
+            self.prompt_time = 0
+            
+            # ask for another prompt
+            another.set(42)
+    =}
+     
+    reaction(shutdown) {=
+        if self.count > 0:
+            print("Average response time: {:.2f} ms".format(self.total_time_in_ms / self.count))
+        else:
+            print("Average response_time: undefined")
+    =}
+}
+
+/*
+ * Starts the GUI and pass the user_input_pout and 
+ *     update_graphics_pin Pipe objects
+ *     to GetUserInput and UpdateGraphics
+ */
+reactor StartGui {
+    preamble {=
+        import gui
+    =}
+        
+    timer t(0, 1 msec);
+    output user_input_pipe;
+    output update_graphics_pipe;
+    
+    reaction(startup) -> user_input_pipe, update_graphics_pipe {=
+        # Starts the gui pygame process
+        user_input_pout, update_graphics_pin = self.gui.start_gui()
+        
+        # Sets the outputs to trigger the initialization of GetUserInput and UpdateGrpahics 
+        user_input_pipe.set(user_input_pout)
+        update_graphics_pipe.set(update_graphics_pin)
+    =}
+    
+    reaction(t) {=
```

Comment:
I don't think this is needed anymore right?

Reply:
It does look like this branch is behind `master` by quite a bit. You would have to merge in master first and then rebuild lfc and/or Eclipse based on the tools you use.